airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -39,25 +39,31 @@ module.exports = function (plop) {
     description: 'Generate an Airbyte Source written in Python',
     prompts: [{type: 'input', name: 'name', message: 'Source name, without the "source-" prefix e.g: "google-analytics"'}],
     actions: [
-        {
```

Comment:
why is this changing?

Reply:
because I forgot to exclude it from the change.